cmd/scheduler.go

```diff
@@ -168,16 +168,16 @@ func loadConfig() {
 }
 
 // This method handles the http request used to start a job.
-func handleHttpFunction(w http.ResponseWriter, r *http.Request) {
+func handleHttpEvent(commandName string) {
 	// Look for the Importer command in the list of commands
 	var newCommand scheduler.Command
 	for _, cmd := range commands {
-		if cmd.Name == "ImqsTool Importer" {
+		if cmd.Name == commandName {
```

Comment:
Don't copy the command into "newCommand". Instead, just store a pointer to the actual command. So you'd want one variable such as `var command *scheduler.Command`
 
Don't mess with StartTime. This is part of the configuration of a command. Also don't go through scheduler.NextRunnable. Just run the command using command.Run().

The function handleHttpEvent is badly named, because it implies that it is going to run on the same thread that handles HTTP events. Instead, it should be named something generic, such as "runCommandNow".